packages/deployment/upgrade-test/upgrade-test-scripts/unreleased-upgrade/actions.test.js

```diff
@@ -27,16 +53,20 @@ test.before(async t => {
   t.context.bundleIds = await installBundles(bundlesData);
 });
 
-test.skip('Open Vaults', async t => {
-  const currentVaults = await agops.vaults('list', '--from', GOV1ADDR);
-  t.is(currentVaults.length, 5);
+test('Open Vaults with auto-provisioned wallet', async t => {
+  const { userAddress } = /** @type {{userAddress: string}} */ (t.context);
+  t.is(await getISTBalance(userAddress), 1);
+
+  const ATOMGiven = 2000;
+  const ISTWanted = 400;
+  // Decompose openVault because follower is broken for auto-provisioned wallets
```

Comment:
Is this comment still accurate?

Reply:
Nope, it got stuck in the fix up, will remove.